lib/middleware/errors.js

```diff
@@ -12,31 +13,27 @@ function canWithstand(error) {
   }
 }
 
-function _renderError(err, req, res, next, noStack) {
-  res.status(err.status || 500);
+function _applicationError(err, req, res, next) {
+  err.status = err.status || 500;
+
   logger.log('error', {'Application error': err.message});
   logger.log('error', {'Error stack': err.stack});
 
-  const errorPath = config.KAT_ERROR_PATH;
-  res.redirect(`${errorPath}/${res.statusCode}`);
+  next(err);
```

Comment:
The idea is that we pass `renderView` to the final `app.use()` in the **server/index.js** file for the application.

It just means that if for one of the apps we wanted to check for something else before rendering the view, we just have to add another `app.use()` and make sure the function passed to it also calls `next(err)`, rather than needing to edit `kmt-utilities` to call another function. It's pretty much in line with the error handling examples in the Express docs.